Mulligan.1.html

```diff
@@ -0,0 +1,174 @@
+<html>
+    <head>
+            <style>
+                 div.padded {  
+      padding-top: 50px;  
+      padding-right: 400px;  
+      padding-bottom: 0.25in;  
+      padding-left: 400px;  
+    }
+            #rcorners2 {
+            border-radius: 25px;
+            border: 10px solid; 
+            width: 200px;
+            height: 200px;
+            color: DodgerBlue;
+            float: left;
+            margin-right: 20px
+            
+                    }
+            #rcorners3 {
+            border-radius: 25px;
+            border: 10px solid; 
+            width: 200px;
+            height: 200px;
+            color: plum;
+            float: right;
+            margin-left: 20px
+                    }   
+            #rcorners4 {
+            border-radius: 25px;
+            border: 10px solid; 
+            width: 200px;
+            height: 200px;
+            color: orange;
+            float: left;
+            margin-right: 20px
+                    }        
+            #rcorners1 {
+            border-radius: 25px;
+            border: 10px solid; 
+            width: 200px;
+            height: 200px;
+            color:red;
+            float: right;
+            margin-left: 20px
+                    }          
+
+                    .main {
+             color: black;
+             font-size: 16px;
+             font-family: Helvetica;
+             
+             }
+
+            .name {
+             color: dodgerblue;
+             font-size: 30px;
+             font-family: Helvetica;
+             
+             }
+            .name2 {
+             color: plum;
+             font-size: 30px;
+             font-family: Helvetica;
+             }
+             .name3 {
+             color:orange;
+             font-size: 30px;
+             font-family: Helvetica;
+             }
+             .name4 {
+             color: red;
+             font-size: 30px;
+             font-family: Helvetica;
+             }
+
+             img.rounded-corners {
+border-radius: 15px;
+}
+
+             .image-container {
+  display: flex;
+  justify-content: center;
+}
+             
+        
+            
+            </style>
+    </head>
+        <body>
+                
+                <!-- get the header image to center on the page -->
+                <div class="image-container"><img src="TMNTHead.png" alt="TMNTHead" style="width:1400px;height:832px;"></div>
```

Comment:
- Alt tags should be descriptive so that the visually impaired know what they're reading.
- width and height should be in the css
- images should be in their own folder `/img`

`<img src-"" alt="Teenage Mutant Ninja Turtles Heads">`